pkg/reconciler/ingress/ingress.go

```diff
@@ -324,10 +340,35 @@ func (r *Reconciler) reconcileRouteDeletion(ctx context.Context, ia *v1alpha1.In
 		return err
 	}
 
+	ingresses, err := r.ingressLister.Ingresses(ia.GetNamespace()).List(labels.Everything())
```

Comment:
Actually, on the label: Shall we filter this list by the `istio` class? We'd want to deal with the SMMR once we remove the last **istio** ingress, right?

Reply:
That makes sense. But the `istio` class (`networking.knative.dev/ingress.class`) is an annotation key not label key. I am wondering that it it difficult (impossible(?)) to list the annotation key.